examples/end-to-end/bookstore/server.go

```diff
@@ -46,39 +45,31 @@ type server struct {
 	Mutex       sync.Mutex // global mutex to synchronize service access
 }
 
-func (s *server) ListShelves(context.Context, *empty.Empty) (*ListShelvesResponses, error) {
+func (s *server) ListShelves(context.Context, *empty.Empty) (*ListShelvesResponse, error) {
 	s.Mutex.Lock()
 	defer s.Mutex.Unlock()
 	// copy shelf ids from Shelves map keys
 	shelves := make([]*Shelf, 0, len(s.Shelves))
 	for _, shelf := range s.Shelves {
 		shelves = append(shelves, shelf)
 	}
-	responses := &ListShelvesResponses{
-		Ok: &ListShelvesOK{
-			ApplicationJson: &ListShelvesResponse{
-				Shelves: shelves,
-			},
-		},
+	response := &ListShelvesResponse{
+		Shelves: shelves,
 	}
-	return responses, nil
+	return response, nil
 }
 
-func (s *server) CreateShelf(ctx context.Context, parameters *CreateShelfParameters) (*CreateShelfResponses, error) {
+func (s *server) CreateShelf(ctx context.Context, parameters *CreateShelfParameters) (*Shelf, error) {
 	s.Mutex.Lock()
 	defer s.Mutex.Unlock()
 	// assign an id and name to a shelf and add it to the Shelves map.
-	shelf := parameters.RequestBody.ApplicationJson
+	shelf := parameters.Shelf
 	s.LastShelfID++
 	sid := s.LastShelfID
 	s.Shelves[sid] = shelf
 
-	responses := &CreateShelfResponses{
-		Ok: &CreateShelfOK{
-			ApplicationJson: shelf,
-		},
-	}
-	return responses, nil
+	response := shelf
+	return response, nil
```

Comment:
I guess this is the same server that's in end-to-end-grpc-gateway (and would get the same updates)